css/mistica-common.css

```diff
@@ -695,3 +705,90 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
 .mistica-card .mistica-card__buttons {
     flex: 1;
 }
+
+/* Accordion */
+
+.mistica-accordion-item {
+    interpolate-size: allow-keywords;
+}
+
+.mistica-accordion-item:after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 1px;
+    background-color: var(--vcolor-divider);
+}
+
+.mistica-accordion-item:last-child:after {
+    display: none;
+}
+
+.mistica-accordion-item__summary {
+    cursor: pointer;
+    min-height: 72px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 16px;
+    margin: 0 -16px;
+    transition: background-color 0.1s ease-in-out;
+}
+
+.mistica-accordion-item__summary-body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 2px;
+}
+
+.mistica-accordion-item__summary-title {
+    font-size: var(--mistica-font-size-3);
+    line-height: var(--mistica-line-height-3);
+    font-weight: var(--mistica-font-weight-3);
+    color: var(--vcolor-textPrimary);
+}
+
+.mistica-accordion-item__summary-subtitle {
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+    font-weight: var(--mistica-font-weight-2);
+    color: var(--vcolor-textPrimary);
+}
+
+.mistica-accordion-item__chevron {
+    transition: transform 0.4s;
+    color: var(--vcolor-accordionChevronClosed);
+}
+
+.mistica-accordion-item[open] > summary > .mistica-accordion-item__chevron {
+    transform: rotate(-180deg);
+    color: var(--vcolor-accordionChevronOpen);
+}
+
+.mistica-accordion-item__summary:hover {
+    background-color: var(--mistica-color-backgroundContainerHover);
+}
+
+.mistica-accordion-item__summary:active {
+    background-color: var(--mistica-color-backgroundContainerPressed);
```

Comment:
default context: `backgroundContainerPressed`
inverse context: `backgroundContainerBrandPressed`

https://www.figma.com/design/KDxPjcTXq2NCvQZjb4FXgi/%F0%9F%94%B8-Accordion-Specs?m=auto&node-id=2430-12861&t=HfNeddBVH2Sd05zU-1

Reply:
should be fixed

Reply:
Weird, I still see default hover

![Screenshot 2025-04-14 at 12 26 40](https://github.com/user-attachments/assets/babfcb0a-b051-467d-bea8-65467b7f2645)

https://mistica-a4g4tyw7j-flows-projects-65bb050e.vercel.app/mistica-css